pkg/apis/kops/v1alpha3/cluster.go

```diff
@@ -240,6 +240,9 @@ type GCESpec struct {
 	NodeInstancePrefix *string `json:"nodeInstancePrefix,omitempty"`
 	// PDCSIDriver is the config for the PD CSI driver.
 	PDCSIDriver *PDCSIDriver `json:"pdCSIDriver,omitempty"`
+
+	// BinariesLocation is the location of the GCE cloud provider binaries.
+	BinariesLocation *string `json:"binaryLocation,omitempty"`
```

Comment:
```suggestion
	BinaryLocation *string `json:"binaryLocation,omitempty"`
```

(Yes, the AWS analogue is misnamed as well)

Reply:
It can be multiple ones in the future.

Reply:
But the json field name is `binaryLocation`. It doesn't make sense having a difference in number like that.

Reply:
probably both should be plural... (both gcp and aws)

Reply:
@upodroid Can you use the plural for the json field also?

Reply:
Isn't the singular form already baked into the AWS API in a released version?